src/src.pro

```diff
@@ -18,8 +18,8 @@
 #    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 5) {
-  error("requires Qt 5.0 or later")
+lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
```

Comment:
:thinking: You know - I am not __that__ certain that `lessThan(...)` was ever documented in the [Qt4.x qmake manual](http://doc.qt.io/qt-4.8/qmake-function-reference.html).